src/main/java/mate/academy/EventManager.java

```diff
@@ -1,19 +1,40 @@
 package mate.academy;
 
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
 public class EventManager {
-    public void registerListener(EventListener listener) {
+    private static final int TIMEOUT = 10;
+    private final CopyOnWriteArrayList<EventListener> listeners = new CopyOnWriteArrayList<>();
```

Comment:
Better to use Set to avoid eventListeners to registed 2 times

Reply:
We have methoud addIdAbsend that checks elements uniquiness